src/sass/style.scss

```diff
@@ -38,6 +36,7 @@
 
 @import "~uswds/src/stylesheets/core/grid";
 @import "~uswds/src/stylesheets/core/utilities";
+@import "~uswds/src/stylesheets/core/base";
 
```

Comment:
So it turns out that we did have to do a little more than update our dependencies. USWDS changed their import order, so base.scss needed access to the `focus` mixin from the utilities.